flocker/node/agents/test/test_ebs.py

```diff
@@ -4,10 +4,32 @@
 Tests for ``flocker.node.agents.ebs``.
 """
 
+from string import lowercase
+from uuid import uuid4
+
+from hypothesis import given
+from hypothesis.strategies import integers, sampled_from, builds
+
+from bitmath import GiB
+
 from twisted.python.filepath import FilePath
 from twisted.trial.unittest import SynchronousTestCase
 
-from ..ebs import AttachedUnexpectedDevice, _expected_device
+from ..ebs import (
+    AttachedUnexpectedDevice, _expected_device,
+    _attach_volume_and_wait_for_device, _get_blockdevices,
+    _get_device_size,
+)
+from ..blockdevice import BlockDeviceVolume
+
+from ....testtools import CustomException
+
+
+# A Hypothesis strategy for generating /dev/sd?
```

Comment:
:grey_question: For bonus points you could also make this generate /dev/sd?? (e.g. /dev/sdaa) or /dev/sd??? I believe thats where the kernel puts >26 SCSI disks. You can't attach that many with ebs, but I'm guessing it'd be tempting to just copy this for other drivers, and richardw@ was working on a system the other day that was way off into the /dev/sdbx range with attached devices.